instagram/src/components/CommentSection/CommentInput.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+const CommentInput = props => {
+    return (
+        <div className='comment-section'>
+            <form onSubmit={props.addNewComment}>
+                <input 
+                className='comment-input'
+                type = 'text'
+                placeholder = 'Add a comment...'
+                value={props.inputText}
+                onChange={props.handleChange}
+                />
+            </form>
+            <i class="fas fa-ellipsis-h"></i>
```

Comment:
Saw the error in your console, make sure you change this to className since we're in react.